Kickstarter-iOS/EmailFrequency.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+public enum EmailFrequency: Int {
+  case daily
+  case individualEmails
+
+  public static let allCases: [EmailFrequency] = [.daily, .individualEmails]
+
+  public static let rowHeight: CGFloat = 45.0
```

Comment:
Curious - I know in the past we used grid values for things, not really applicable here? Not sure if designs still come in grids?

Reply:
Ah ok I didn't know about the grid values, now I do! Will update